src/class/api.submissions.php

```diff
@@ -503,7 +503,8 @@ public function processPOST ()
         if (!$_INI['paths']['data_files']) {
             // There's no way we can do anything with the results, so fail.
             $this->API->aResponse['errors'][] = 'No data file path configured. Without this path, this LOVD API is not able to process files. ' .
-                'Please contact the admin to configure the data file path: ' . $_SETT['admin']['name'] . ' <' . $_SETT['admin']['email'] . '>.';
+                'Please contact the admin to configure the data file path: ' . $_SETT['admin']['name'] . ' <' .
+                str_replace(array("\r\n", "\r", "\n"), '>, <', trim($_SETT['admin']['email'])) . '>.';
```

Comment:
This string replacement:
~~~php
str_replace(array("\r\n", "\r", "\n"), '>, <', trim($_SETT['admin']['email'])) . '>.'
~~~
is done 4 times in the same (rather big) function. I think it's better to put it in a variable somewhere at the top to make it more maintainable.

Reply:
Makes sense... Implemented in 7198a5b.